cg/apps/sequencing_metrics_parser/models/bcl2fastq_metrics.py

```diff
@@ -1,56 +1,54 @@
-from typing import List
+from typing import List, Optional
 from pydantic import BaseModel, Field
 
 
 class ReadMetric(BaseModel):
-    """
-    Represents a set of metrics generated by bcl2fastq for a read in the sequencing process.
-    """
+    """Metrics for a read for a sample on a tile in a lane."""
 
-    yield_: int = Field(..., alias="Yield", ge=0)
-    yield_q30: int = Field(..., alias="YieldQ30", ge=0)
-    quality_score_sum: int = Field(..., alias="QualityScoreSum", ge=0)
+    read_yield: int = Field(..., alias="Yield", ge=0)
+    read_yield_q30: int = Field(..., alias="YieldQ30", ge=0)
+    read_quality_score_sum: int = Field(..., alias="QualityScoreSum", ge=0)
 
 
 class DemuxResult(BaseModel):
-    """
-    Represents the result of the bcl2fastq demultiplexing process for a given sample.
-    """
+    """Metrics for a sample on a tile in a lane."""
 
     sample_id: str = Field(..., alias="SampleId", min_length=1)
-    number_reads: int = Field(..., alias="NumberReads", ge=0)
-    yield_: int = Field(..., alias="Yield", ge=0)
-    read_metrics: List[ReadMetric] = Field(..., alias="ReadMetrics")
+    tile_sample_reads: int = Field(..., alias="NumberReads", ge=0)
+    tile_sample_yield: int = Field(..., alias="Yield", ge=0)
+    tile_sample_read_metrics: List[ReadMetric] = Field(..., alias="ReadMetrics")
+
+
+class Undetermined(BaseModel):
+    """Metrics for reads without a sample id within a tile of a lane."""
```

Comment:
Maybe UndeterminedReads or something? I would have a hard time understanding what Undetermined represents.

Reply:
Did not realise these models can be renamed as longs as they are aliased correctly 👍 Yes, something like that sounds better

Reply:
Renamed to `UndeterminedTileReads`